packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.js

```diff
@@ -182,6 +190,19 @@ export default class QuanticResultQuickview extends LightningElement {
     });
   }
 
+  lastElementOnFocus() {
+    // @ts-ignore
+    this.getCloseButton?.focus();
+  }
+
+  get headerElement() {
+    return this.template.querySelector('c-quantic-result-link');
+  }
+
+  get closeButton() {
+    return this.template.querySelector(`.slds-button.slds-button_icon`);
+  }
+
```

Comment:
I think here it makes sense to have these two calls because there's more than 1.
In other components I think if there's just 1 focus target you don't need to make a separate method.

Reply:
methods changed to `setFocusToHeader` & `setFocusToTop`